config.go

```diff
@@ -122,6 +124,31 @@ func (h hypervisor) image() string {
 	return h.Image
 }
 
+func (h hypervisor) kernelParams() []vc.Param {
```

Comment:
Please could you add some tests to `TestHypervisorDefaults()` for this new function.